### PR TITLE
Meeday/feat/dashboard api

### PR DESCRIPTION
**Pull request by meeday**

### CODE REVIEW

File: src/components/pages/NewRisk/NewRisk.js [hunk 1]
```diff
@@ -62,81 +62,99 @@ function NewRisk(props) {
   const [riskScore, setRiskScore] = useState(2);
   const [riskLocation, setRiskLocation] = useState(mapCentre);
 
+  const getLocation = async () => {
+    const { data } = await ProjectService.getProject(
+      "5f53f1adeb1bd77a1004ba12"
+    );
+    console.log(data.project.location);
```
Comment (JoeDodgson) [inline src/components/pages/NewRisk/NewRisk.js:69]: Remove console.log

File: src/components/pages/NewRisk/NewRisk.js [hunk 2]
```diff
@@ -31,22 +31,22 @@ const mapContainerStyle = {
...
 
 // ******* projectID will need to be retrieved
 const projectId = "5f52a6cc0c5677512c956ded";
 
 // ******* mapCentre will need to be retrieved
 let mapCentre = {
   lat: 52.475,
-  lng: -1.900,
+  lng: -1.9,
 };
```
Comment (JoeDodgson) [inline src/components/pages/NewRisk/NewRisk.js:43]: Remove this variable, instead pass this object into the useState on line 63

File: src/components/pages/NewRisk/NewRisk.js [hunk 3]
```diff
@@ -62,81 +62,99 @@ function NewRisk(props) {
   const [riskScore, setRiskScore] = useState(2);
   const [riskLocation, setRiskLocation] = useState(mapCentre);
```
Comment (JoeDodgson) [inline src/components/pages/NewRisk/NewRisk.js:63]: Pass object in instead of mapCentre:
`{
  lat: 52.475,
  lng: -1.9,
}`

**Review by JoeDodgson — CHANGES_REQUESTED** [review]

File: src/components/pages/NewRisk/NewRisk.js (hunk 1 above)
Comment (meeday) [reply src/components/pages/NewRisk/NewRisk.js]: console log removed

File: src/components/pages/NewRisk/NewRisk.js (hunk 2 above)
Comment (meeday) [reply src/components/pages/NewRisk/NewRisk.js]: variable removed

File: src/components/pages/NewRisk/NewRisk.js (hunk 3 above)
Comment (meeday) [reply src/components/pages/NewRisk/NewRisk.js]: object passed through instead of variable

**Review by JoeDodgson — APPROVED** [review]

**Pull request merged**